recipe-server/normandy/base/middleware.py

```diff
@@ -24,3 +26,31 @@ class RequestSummaryLogger(MiddlewareMixin, OriginalRequestSummaryLogger):
     Adapt mozilla_cloud_services_logger's request logger to Django 1.10 new-style middleware.
     """
     pass
+
+
+class ConfigurableRemoteUserMiddleware(RemoteUserMiddleware):
+    """
+    Makes RemoteUserMiddleware customizable via settings.
+    """
+
+    @property
+    def header(self):
+        """
+        Name of request header to grab username from.
+
+        This will be the key as used in the request.META dictionary. To
+        reference HTTP headers, this value should be all upper case and
+        prefixed with "HTTP_".
+        """
+        return settings.OIDC_REMOTE_AUTH_HEADER
```

Comment:
i would rather specify the header as `HTTP_OIDC_CLAIM_USER_EMAIL` than `Oidc-Claim-User-Email`, because that's how I specify it in openresty lua. It sounds like that would work either way though, and this would allow for more flexible input.